net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml

```diff
@@ -7,13 +7,20 @@
             <default>0</default>
             <Required>Y</Required>
         </enabled>
+        <disablelogin type="BooleanField">
+            <default>0</default>        
+            <Required>N</Required>
+        </disablelogin>         
         <interface type="InterfaceField">
             <Required>N</Required>
-            <Multiple>N</Multiple>
+            <Multiple>Y</Multiple>
         </interface>
-        <httpport type="PortField">
+        <httpport type="IntegerField">
```

Comment:
I'm not merging this. Please add a checkbox..

Reply:
fixed - but not with yet another checkbox.
There is only one port in the model now, used for either HTTP (when no cert is selected) or for HTTPS traffic (when cert is selected). The unused option gets disabled in ntopng.conf